src/components/EncounterPreviewContainer/EncounterPreviewContainer.jsx

```diff
@@ -1,5 +1,28 @@
 import PropTypes from 'prop-types';
 import './EncounterPreviewContainer.css';
+import EncounterPreviewCard from '../EncounterPreviewCard/EncounterPreviewCard';
+
+const EncounterPreviewContainer = ({ encounters }) => {
+  const encounterPreviews = encounters && encounters.map(encounter => (
```

Comment:
Ah yeah, I actually _had_ to wherever I was mapping. I kept getting errors because it was trying to map before the state updated. So, I set the check in there and lo and behold, it works! 😂